.env

```
@@ -186,3 +186,18 @@ SQUID_GQL_PORT=4352
 # Archive gateway host (Should not be set in local development)
 # For running a production storage-squid instance uncomment the following line (to use the subsquid hosted archive)
 # SQUID_ARCHIVE_GATEWAY_URL=${CUSTOM_ARCHIVE_GATEWAY_URL:-https://v2.archive.subsquid.io/network/joystream}
+
+# =====================================================================================
+## Cloud storage provider configuration
+
+# Enable or disable the cloud storage provider
+ENABLE_STORAGE_PROVIDER=true
+
+## Specify the cloud storage provider to use:
+CLOUD_STORAGE_PROVIDER_NAME=aws
+
+# AWS S3 (replace with your own configuration and credentials)
+AWS_ACCESS_KEY_ID=test-key
+AWS_SECRET_ACCESS_KEY=test-secret
```

```suggestion
AWS_ACCESS_KEY_ID=test
AWS_SECRET_ACCESS_KEY=test
```

To enable the creation of pre-signed URLs for S3 buckets, please set both `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY` to the value `“test”` Our pre-signed URL signature verification algorithm validates the pre-signed URL and its expiration. - note from https://docs.localstack.cloud/user-guide/integrations/aws-cli/#localstack-aws-cli-awslocal